src/AppInstallerCLICore/Core.cpp

```diff
@@ -172,6 +172,10 @@ namespace AppInstaller::CLI
             AICLI_LOG(CLI, Error, << "Operation blocked by Group Policy: " << policy.RegValueName());
             context.Reporter.Error() << Resource::String::DisabledByGroupPolicy(policy.PolicyName()) << std::endl;
             return APPINSTALLER_CLI_ERROR_BLOCKED_BY_POLICY;
+        }
+        catch (...)
+        {
+            return Workflow::HandleException(context, std::current_exception());

```

Comment:
nit: Workflow::HandleException handles GroupPolicyException above. Btw I don't have preference on whether we would list GroupPolicyException above explicitly or let HandleException handle all non-command exceptions.

Reply:
I'm in favor of having explicit handling in this case, since the message for Group Policy is quite clear about what is happening